csharp/test/Apache.Arrow.Tests/ArrowArrayTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Apache.Arrow.Tests.Fixtures;
```

Comment:
Could you add a license header?

Reply:
Thank you for your comment!
I added the license header.